src/app/app_states.rs

```diff
@@ -261,6 +261,7 @@ impl AppState {
                 };
 
                 let mut base_events: Vec<AppEvent> = vec![
+                    ExitApp(EventKeyBinding::new(vec![key_bindings.main_menu.exit, key!(ctrl-c)], "Exit", Some("Exit"))),
```

Comment:
@Julien-cpsn 
>On platforms such as Windows or many Linux distros, ctrl-c is used to copy text from the terminal. Imagine that I want to manually copy the request URL onto my clipboard, with your code it will exit the app (without even copying the text).

Oops, you are right, I'm still not used to think of this as a multi-platform bin.
wouldn't this be enough? 

```suggestion
                    ExitApp(EventKeyBinding::new(vec![key_bindings.main_menu.exit], "Exit", Some("Exit"))),
```